### PR TITLE
Issue #1 Displaying tags

### PR DESCRIPTION
**Pull request by OnalennaRammapatsi**

### CODE REVIEW

File: database/index.js
Comment (Renzo-codespace) [inline database/index.js:1]: The mongo connection string should NEVER be hardcoded into the project.
It needs to be imported from a .env file, which should be included in the gitignore file.

All functions which use the mongo client should also have client.close() after all db interactions have been done.

Comment (OnalennaRammapatsi) [reply database/index.js:1]: Please marge it, the person working on .env file wants to work from what I already have.

Comment (Renzo-codespace) [reply database/index.js:1]: Please note that committing sensitive information, like connection strings, is a security risk.
Not running client.close() will cause unexpected behaviour in the application.

**Review by Renzo-codespace — APPROVED** [review]
I've stated my concerns in the pull request conversation.

**Review by ElzabeEls — APPROVED** [review]
Approved

**Pull request merged**